filters/gender_bias/test.json

```diff
@@ -0,0 +1,73 @@
+{
```

Comment:
Please format the JSON file as indentation is not proper.

Reply:
Fixed the identation.

Reply:
Hi @aadesh11, thank you for reading the code and providing valuable feedback, which resulted in a code improvement!